.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         run: |
           sudo apt-get install pandoc
           pip install --upgrade pip setuptools
-          pip install -r requirements/requirements_all.txt
+          pip install -r requirements/dev.txt -r requirements/docs.txt
```

Comment:
Note for later could be to consider having the `Build documentation` step in separate build / pipeline (whatever github actions naming convention is) so it uses its own dependency. 

Just conscious that providing more than one `-r` argument may lead to nondeterministic dependency resolution.

Reply:
This makes sense, probably one for a future PR.